src/main/java/fruitstore/FruitShopServeImpl.java

```diff
@@ -0,0 +1,29 @@
+package fruitstore;
+
+import java.util.List;
+import java.util.Map;
+import records.parser.RecordsSaver;
+import records.parser.RecordsSaverCommaSeparated;
+import records.reader.RecordsFileReader;
+import records.reader.RecordsReader;
+import records.writer.RecordsFileWriter;
+import records.writer.RecordsWriter;
+
+public class FruitShopServeImpl implements ShopServe {
+    private RecordsReader recordsReader;
+    private RecordsSaver saver;
+    private RecordsWriter writer;
+
+    public FruitShopServeImpl() {
+        recordsReader = new RecordsFileReader();
+        saver = new RecordsSaverCommaSeparated();
+        writer = new RecordsFileWriter();
+    }
```

Comment:
The code should not be implementation dependent
```suggestion
    public FruitShopServeImpl(
    RecordsReader recordsReader,
    RecordsSaver saver,
    RecordsWriter writer) {
    
        this.recordsReader = recordsReader;
        this.saver = nsaver;
        this.writer = writer;
    }
```